webfront/views/set.py

```diff
@@ -57,10 +37,7 @@ def get(
         )
         general_handler.modifiers.register(
             "alignments",
-            get_set_alignment,
-            type=ModifierType.REPLACE_PAYLOAD,
-            many=True,
-            serializer=SerializerDetail.SET_ALIGNMENT,
+            get_deprecated_response("The 'alignments' modifier was removed with InterPro 98. Contact interhelp@ebi.ac.uk if you are interested in this data.")
```

Comment:
If we encourage users to contact us, we may need to keep the data up-to-date internally to be able to share... and I'd rather not. ;)

```suggestion
            get_deprecated_response("Profile-profile alignments have been permanently removed in InterPro 98.0.")
```